translations/messages.de.yaml

```diff
@@ -30,6 +30,9 @@ ngsite.search.no_results.reduce_keywords: 'Reduzieren Sie die Anzahl der Keyword
 ngsite.collected_info.success_text: 'Wir danken Ihnen für Ihr Feedback.'
 ngsite.collected_info.button.send: 'Formular senden'
 ngsite.collected_info.information_collected: 'Die folgenden Informationen wurden gesammelt'
+ngsite.collected_info.given_name.attribute: 'given-name'
```

Comment:
Why is this in translation?

Reply:
these are predefined autocomplete tokens

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

should I just write them in the form?
`{{ form_row(form.sender_first_name, { ngparams: { autocomplete: 'given-name' } })}}`